src/VisualStudio/Core/Test.Next/Services/LspDiagnosticsTests.cs

```diff
@@ -378,7 +378,7 @@ static InProcLanguageServer CreateLanguageServer(Stream inputStream, Stream outp
             {
                 var protocol = ((TestWorkspace)workspace).ExportProvider.GetExportedValue<LanguageServerProtocol>();
 
-                var languageServer = new InProcLanguageServer(inputStream, outputStream, protocol, workspace, mockDiagnosticService, clientName: "RazorCSharp");
+                var languageServer = new InProcLanguageServer(inputStream, outputStream, protocol, workspace, mockDiagnosticService, clientName: null);
```

Comment:
Why did this change?

Reply:
I talked to David and it's unclear why clientName had a value of RazorCSharp in the first place. I believe these tests should just be using the regular C# language client.